Kubernetes/windows/helper.v2.psm1

```diff
@@ -407,7 +413,12 @@ function DownloadCniBinaries($NetworkMode, $CniPath)
     DownloadFile -Url  "https://github.com/$Global:GithubSDNRepository/raw/$Global:GithubSDNBranch/Kubernetes/flannel/l2bridge/cni/flannel.exe" -Destination $CniPath\flannel.exe
     DownloadFile -Url  "https://github.com/$Global:GithubSDNRepository/raw/$Global:GithubSDNBranch/Kubernetes/flannel/l2bridge/cni/host-local.exe" -Destination $CniPath\host-local.exe
 
-    if ($NetworkMode -eq "l2bridge")
+    if ($Global:Cri -eq "containerd")
+    {
+        DownloadFile  "https://github.com/microsoft/windows-container-networking/releases/download/v0.2.0/windows-container-networking-cni-amd64-v0.2.0.zip" -Destination "$env:TEMP\windows-container-networking-cni-amd64-v0.2.0.zip"
+        Expand-Archive -Path "$env:TEMP\windows-container-networking-cni-amd64-v0.2.0.zip" -DestinationPath $CniPath -Force
+    }
+    elseif ($NetworkMode -eq "l2bridge")
```

Comment:
Todo: This check should do the same as previous - if docker, then download it from GitHub.com/containernetworking/plugins repo

Reply:
Does what is advertised.